pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+### Description
+<!--
+_Using one or more sentences, describe the proposed changes and the reason for making them._
+-->
+
+### Related JIRA Issue(s)
+<!--_Please provide the URL(s) for any JIRA issues related to this PR._-->
+
+### Review App URL(s) 
+http://__BRANCH_NAME_HERE__.review.ensembl.org
+
+
+### Knowledge Base
+<!--
+_If there is any new concept which are being implemented then provide link(s) to the Knowledge base (Link to Documentation, Blog etc )_
```

Comment:
`which are being implemented` -> `which is being implemented`. Or `If the PR introduces new concepts, then...`